docs/examples/04_backends_vrep/01_simulation_with_vrep.rst

```diff
@@ -16,7 +16,7 @@ The first step is just connect to the simulator and verify the connection
 is working.
 
 Copy and paste the following example into any Python environment
-(a standalong script, a CAD environment, etc) and run it, you should
+(a standalone script, a CAD environment, etc) and run it, you should
 see the output ``Connected: True`` if everything is working properly:
 
 .. code-block:: python
```

Comment:
I want to comment on line 44, but it's not clear to me how to do that... Anyway, the sentence on line 44 reads awkwardly to me... I'm not sure about this extra "of them" that appears.

Reply:
you're totally right, fixed it on https://github.com/compas-dev/compas_fab/pull/169/commits/fd8e654dc18c9dd81f6b66b8463fc8c4f4bd5523